### ISSUE TITLE
While installing the drive should be added when asked for the partitions

### ISSUE THREAD

**Issue by xamafe** [BlackArch/blackarch-installer#92]
Let's begin with a checklist: Replace the empty checkboxes [ ] below with checked ones [x] accordingly. -->

- [X] I have searched open and closed issues for duplicates
- [X] I am submitting a bug report for existing functionality that does not work as intended

----------------------------------------

### Bug description

While installing the full iso image to a bare metal machine, the user is asked, which partitions are EFI/SWAP/ROOT ...
The output states the chosen disk like this: /dev/nvme0n1X 

The X looks like the user just has to add the missing part, like "p1", "p2" and so on.
But the system failes after 3 more steps, when trying to mkfs.fat because it can not open "p1".

### Steps to reproduce

* Install normally
* Use EFI installation
* choose different partitions
* give p1 to p3 as partitions, when asked where the partitions are located

**Actual result:** 
The mkfs.fat fails with the error it could not open "1" or "p1"

**Expected result:** 
The system should check, if the full path to the device is given, or if its just the partition and fill the drive accordingly

### Info for developers

none needed, I guess?

**Comment by xamafe** [comment]
One can argue if this is a bug or a feature request, but after the output states by the X, that only the missing part is needed, I think we can call this a bug :-)

**Comment by D0bledore** [comment]
Yes I agree. But for anyone who is trying to install it blackarch:

here is the solution (which should be obvious) 

when the blackarch-installer asks which partition, give the **full path** (e.g. /dev/nvme0n1p1  = for EFI and so on for the others.)
